arbor/include/arbor/cable_cell.hpp

```diff
@@ -64,7 +68,7 @@ struct cable_probe_point_info {
 // Sample value type: `double`
 // Sample metadata type: `mlocation`
```

Comment:
Should this read `locset` instead?

Reply:
Possibly surprisingly, no: the locset is evaluated to some collection of (thingified) locations; there will be a membrane voltage probe at each location, and that location will constitute the probe-specific metadata.